### ISSUE TITLE
1.19.2 V1.1.2 Drill will mine blocks such as Cargo Manager, fluid pipes, etc

### ISSUE THREAD

**Issue by Masonatorr** [CreeperHost/StevesCarts2#37]
I have the Cargo Manager and Liquid Manager on either side of an Advanced Detector Rail, with a rail line on one side and a dead end on the other. I have had issues where:
- The drill will mine whatever blocks comprise the dead end (including Cargo Managers)
- The cart will drill downwards into the dead end to reach the target Y, fail to place a rail, and continue to drill *straight* downwards all the way to the target Y.
- The drill will mine some of the pipes that I am using to fill the Liquid Manager

Right now, using this mod to make a mining cart is unfortunately more trouble than it's worth.

I propose having a default filter where the cart will not drill through blocks that aren't generated with the world. or at least prevent it from drilling into a wall when the detector rail is touching said wall.

**Comment by gigabit101** [comment]
Yup the drill will mine anything, You can use the module toggler block to disable the drill in locations, This is intended for now but might be something we can look into improving
